CONTRIBUTING.md

```diff
@@ -92,6 +94,9 @@ Once you have babel running in watch mode, you can start making changes to parse
 *Question*: How do I deploy my forked version on my servers?<br/>
 *Answer*: In your `package.json`, update the `parse-server` dependency to `https://github.com/MY_USERNAME/parse-server#MY_FEATURE`. Run `npm install`, commit the changes and deploy to your servers.
 
+*Question*: How do I deploy my forked version using docker?<br/>
+*Answer*: In your `package.json`, update the `parse-server` dependency to `https://github.com/MY_USERNAME/parse-server#MY_FEATURE`. Make sure the `npm install` step in your `Dockerfile` is running under non-privileged user for the ``npm run prepare`` step to work correctly. For official node images from hub.docker.com that non-privileged user is `node` with `/home/node` working directory.
```

Comment:
To not reference a potentially real GitHub user here, please change to:

`https://github.com/[username]/parse-server#[branch/commit]`